### ISSUE TITLE
Create custom in-progress message for Hearing aid order form

### ISSUE THREAD

**Issue by peggygannon** [department-of-veterans-affairs/va.gov-team#10462]
## User story
We need to create a custom in-progress message for the Hearing aid application to account for users who don't think of this an as "application" where they are applying for something.  
This custom message will replace the current hard-coded in-progress message that current exists for form. 

## Acceptance criteria
- [x] Peggy works with BAM team to write custom in-progress message for the Hearing aid order form app only. 
- [ ] Since this is a custom message, this won't apply globally across all forms.  The BAM-2 team should make design and/or the design council aware of this custom hard-coded message is being used for this form only.

**Comment by peggygannon** [comment]
here is screenshot of the current hard-coded in-progress message. We need to update message to account that the Hearing aid and Battery online order form is not an "application" and users aren't "applying" for anything. 

For future work, I would love to update the language in this message globally, as this wasn't created by a content person. 

![Screen Shot 2020-06-05 at 5.00.18 PM.png](https://images.zenhubusercontent.com/59b0414bb0222d5de476aa22/b320daae-ff2f-44f2-8aa1-223e7b2a28e5)

**Comment by peggygannon** [comment]
@rileyorr @rebeccawalsh @mr0sari0 

Hello, please see below updated content for the in-progress message. A couple of things: 

--I tightened it up and removed the line about "your application was last saved on..." as I'm not sure the value of communicating that to the user? I didn't write these in-progress messages, so I'm not sure if there is a reason that the the "last saved" line is included.
@mr0sari0 , please let me know if there is a dev reason that this line should be in the message. 
 

Please let me know your thoughts on the below: 

**Your order is in progress**
Your order for hearing aid batteries and accessories (Form 2346A) is in progress. You can continue your order now, or come back later to finish. This order will expire on [Month, date, year].

**Comment by rileyorr** [comment]
Thanks for looking into this, @peggygannon! I agree with the proposed content change. We're removing the bottom line "Please complete the 2346 form to apply for ordering hearing aid batteries and accessories." as well, right?

**Comment by peggygannon** [comment]
Hi, @rileyorr Actually, that is a good question... is there a button at the bottom of this message that will take a user back to their online form? I don't see one in the screenshot above, so not sure if part of the message is cut off. 

If not, then i will want to tweak my above message a bit..

**Comment by rileyorr** [comment]
@peggygannon I just grabbed a screenshot of what the in-progress alert currently looks like on [the form's introduction screen](https://staging.va.gov/hearing-aid-batteries-and-accessories/introduction) (signed in as test user 1) and we do have buttons that will either take the user back into the form or start a new form. I think we'll want to update those buttons to say "order" instead of "application" if possible, but I think @mr0sari0 previously told me those buttons can't be edited. What do you think? 

<img width="786" alt="Screen Shot 2020-06-19 at 12 14 33 PM" src="https://user-images.githubusercontent.com/46329147/85155207-7898c280-b226-11ea-8e58-2e5427dfa64d.png">

**Comment by mr0sari0** [comment]
@rileyorr I was just about to grab a screenshot but looks like you beat me to it. Currently the button messages are not able to be changed but I have a [PR for adding custom content to the form system](https://github.com/department-of-veterans-affairs/vets-website/pull/13016).

**Comment by mr0sari0** [comment]
@peggygannon I would defer to @cvalarida and @Clark-W-Griswold on if devs have a need for the "last saved" line but I personally don't see a need for it. I think the only reason to keep this line is if the Veteran found the time and date of their last saved application helpful to them.

**Comment by peggygannon** [comment]
@rileyorr @mr0sari0 ok thank you for grabbing a screenshot showing the buttons. If we can't update the current button language, then I've tweaked the message slightly to tie the message to the button language.  

**Your order is in progress**
Your order for hearing aid batteries and accessories (Form 2346A) is in progress. You can continue your application now, or come back later to finish. This order will expire on [Month, date, year].

{buttons} 
Continue your application
Start a new application

**Comment by rileyorr** [comment]
@peggygannon Awesome, thank you for the alternative message. Hopefully Mahariel's PR gets approved so we can customize the buttons. 

@mr0sari0 did you get a chance to confirm whether or not the "last saved" line is required?

**Comment by mr0sari0** [comment]
@rileyorr I have not talked to @cvalarida  or @Clark-W-Griswold yet. I'll ping them about this.

As far as PR approval timelines, this is a platform level change, so I am waiting for @cvalarida and @Clark-W-Griswold to approve or propose recommended changes.  I'll follow up with them about this as well.

**Comment by Clark-W-Griswold** [comment]
@peggygannon @mr0sari0 

I think it should be fine to remove that text. 

I did get a little confused around the text 
```
Your order is in progress
Your order for hearing aid batteries and accessories (Form 2346A) is in progress. You can continue your application now, or come back later to finish. This order will expire on [Month, date, year].
```

I was unsure if I had a submitted order in progress or if my application to order was in progress.

**Comment by rileyorr** [comment]
@Clark-W-Griswold we don't want users thinking either of those things! The word 'application' confused our participants during usability testing because they aren't actually applying for the benefit of receiving hearing aid batteries/accessories. They're already eligible for the benefit and are placing orders for these items. 

What about:
> Your order form is in progress
Your order form for hearing aid batteries and accessories (Form 2346A) is in progress. You can continue placing your order now, or come back later to finish. This order form will expire on [Month, date, year].

**Comment by peggygannon** [comment]
@rileyorr @Clark-W-Griswold @mr0sari0 Hello! sorry if you were waiting on me for this... I tightened it up even more. 

Do we know if we can change the button language? It would be preferable to have the buttons tie in more closely with the message (order vs. applications). Let me know.. 

**Your order form is in progress**

You can continue placing your hearing aid batteries and accessories (Form 2346A) order now, or come back later to finish. This order will expire on [Month, date, year].

{buttons}
Continue your application
Start a new application

**Comment by rileyorr** [comment]
Hi @peggygannon! No worries at all! I just reached out to @mr0sari0 and it is possible to customize the button's language. So we will be able to put 'Continue your order' and 'Start a new order'. 

Would you like us to use the content you most recently provided in addition to the customized buttons that will tie in more closely with the message?

**Comment by mr0sari0** [comment]
Hi @rileyorr and @peggygannon, I'm happy to announce that the PR for using customText in the forms library was merged today.

**Comment by mr0sari0** [comment]
Hi @peggygannon , I have created another PR [here](https://github.com/department-of-veterans-affairs/vets-website/pull/13425) to capture your last content update.

Here are some screenshots of how it looks.

<img width="887" alt="Screen Shot 2020-07-09 at 12 00 31 PM" src="https://user-images.githubusercontent.com/12755283/87076881-56281280-c1f0-11ea-80ea-740f74ab8728.png">

Where it says (MDOT), that will be updated to "(Form 2346A)" once @camerontesterman makes the update on vets-api.

<img width="827" alt="Screen Shot 2020-07-09 at 2 29 06 PM" src="https://user-images.githubusercontent.com/12755283/87077084-98515400-c1f0-11ea-8f7a-32221b66e658.png">

<img width="1072" alt="Screen Shot 2020-07-09 at 2 30 15 PM" src="https://user-images.githubusercontent.com/12755283/87077164-b8811300-c1f0-11ea-9da1-012b753f0634.png">

**Comment by peggygannon** [comment]
thanks, @mr0sari0 One update for the 2nd screenshot. there is an extra "order" in the first sentence of the message. Can you please update the first sentence so it reads: 

Your order for hearing aid batteries and accessories (MDOT) has been saved.

**Comment by mr0sari0** [comment]
> thanks, @mr0sari0 One update for the 2nd screenshot. there is an extra "order" in the first sentence of the message. Can you please update the first sentence so it reads:
> 
> Your order for hearing aid batteries and accessories (MDOT) has been saved.

@peggygannon That extra "order" is where the application type is. Typically, it would say "Your (whatever the formBenefit is) application has been saved".  I can remove the application type for just MDOT or should I remove the application type across the board?

**Comment by peggygannon** [comment]
Hi, @mr0sari0 OK, thanks for the clarification. Can we remove the "order" from the first part of the sentence then? So it would be:  

Your hearing aid batteries and accessories (MDOT) order has been saved.

**Comment by leahkeeler** [comment]
@mr0sari0 Can this be closed? I see the code has been merged Thanks
